recipes/MobileSAM/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "mobilesam" %}
+{% set version = "0.0.0.20231030.12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
```

```suggestion
{% set version = "1.0.0.20231030.12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
```
they seem to call their version "1.0"